roles/kubernetes/master/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -19,7 +19,7 @@ nodeRegistration:
 {% if inventory_hostname in groups['kube-master'] and inventory_hostname not in groups['kube-node'] %}
```

Comment:
should kubespray phase away from using the master term in general?

Reply:
Pretty much yeah, we have something in our pipe but it's a big change :)